### DISCUSSION TITLE
Import does not work like documented

### DISCUSSION THREAD

**Discussion by soheilhasanjani** [tailwindlabs/tailwindcss#10031] [Help]
Hello, I created a project with create react app and added tailwindcss, and according to the document, I used the following method to add the css file:

global.css
```@import "tailwindcss/base";
@import "tailwindcss/components";
@import "tailwindcss/utilities";

@layer base {
  @import "./base.css";
}
```

base.css
```
html {
  @apply font-iran-sans-x;
}
```

postcss.config.js
```
module.exports = {
  plugins: {
    "postcss-import": {},
    tailwindcss: {},
    autoprefixer: {},
  },
};
```

how to fixed it ?

**Comment by adamwathan** [comment]
Try this:

```css
@import "tailwindcss/components";
@import "tailwindcss/utilities";

@import "./base.css" layer(base);
```

The `postcss-import` plugin will only process `@import` statements properly that are at the root of your CSS and only if they are the first rules in the file.

**Reply by soheilhasanjani** [reply]
thanks . It's normal that I add a custom class, but inside vsccode it doesn't suggest when autocomplete

**Comment by soheilhasanjani** [comment]
I wrote something like that, but most of the classes don't work anymore
```
@import "./base/index.css" layer(base);
@import "./components/index.css" layer(components);
@import "./utilities/index.css" layer(utilities);
```
